docs/hello_world.rst

```diff
@@ -0,0 +1,228 @@
+    experiment_server_cmd = 'experiment_server.py'
+    
+    # The designated experiment synchronization peer
+    sync_host = 127.0.0.1
+    sync_port = __unique_port__
+    
+    # How long we should wait for experiment nodes to connect
+    sync_experiment_start_delay = 1
+
+    # The command which will handle post processing
+    post_process_cmd = 'post_process_dispersy_experiment.sh'
+
+    # Run python in a virtual environment
+    use_local_venv = FALSE
+
+Alternatively, if you want to run your first experiment on the DAS5:
+
+.. code-block:: None
+
+    ## hello_world_das5.conf ##
+    
+    # The name for our experiment (cosmetic only)
+    experiment_name = "hello_world"
+
+    # The scenario file to run
+    scenario_file = 'hello_world.scenario'
+    
+    # The amound of physical machines to reserve (never more than 20)
+    das4_node_amount = 5
+    
+    # The amount of processes to run over all nodes
+    # These will be evenly mapped over the available nodes (in this case 1 per node)
+    das4_instances_to_run = 5
+    
+    # The time before we automatically hard-kill this experiment
+    das4_node_timeout = 90
+
+    ## Don't change the following settings, unless you are 100% sure you know what you are doing
+
+    # The DAS5 deployment scripts
+    local_setup_cmd = 'das4_setup.sh'
+    local_instance_cmd = 'das4_reserve_and_run.sh'
+    das4_node_command = "launch_scenario.py"
+
+    # Run a Dispersy tracker for peer discovery
+    tracker_cmd = 'run_tracker.sh'
+    tracker_port = __unique_port__
+
+    # The experiment setup script to use
+    experiment_server_cmd = 'experiment_server.py'
+    
+    # The designated experiment synchronization peer's port
+    sync_port = __unique_port__
+    
+    # How long we should wait for experiment nodes to connect
+    sync_experiment_start_delay = 1
+
+    # The command which will handle post processing
+    post_process_cmd = 'post_process_dispersy_experiment.sh'
+
+    # Run python in a virtual environment
+    use_local_venv = TRUE
+    
+    # Use systemtap (for debugging)
+    with_systemtap = false
+    
+The scenario file
+-----------------
+Now that we have instructed Gumby how to set up our environment, we can write the file in charge of generating events: the scenario (.scenario) file.
+Consider the following ``hello_world.scenario``:
+
+.. code-block:: python
+
+    ## hello_world.scenario ##
+    # With this we tell Gumby to load the DispersyModule, which takes care of providing a Dispersy instance for us
+    # If you want extended Tribler functionality, you should use gumby.modules.tribler_module.TriblerModule instead
+    &module gumby.modules.dispersy_module.DispersyModule
+    
+    # This tells Gumby to load our hello_world_module.py file's HelloWorldModule class
+    &module experiments.hello_world.hello_world_module.HelloWorldModule
+    
+    # At 0 seconds, make sure our HelloWorldCommunity does not communicate with the outside world
```

Comment:
I would say: At 0 seconds into the experiment ... (also for the subsequent comments)